lib/puppet/application/filebucket.rb

```diff
@@ -43,6 +43,9 @@ def help
   path to this argument; you are not restricted to restoring the content
   to its original location.
 
+diff:
+  Print a diff between two md5 in the filebucket or between a md5 and
+  its matching file
```

Comment:
The comment says `md5` but implementation seems to use the "current" checksum type: `"#{@rest_path}#{@checksum_type}/#{suma}"`, for example if `Puppet[:digest_algorithm] = 'sha256'`